<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configure_pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrate_pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TESTING OBJ DETECTION</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
